### ISSUE TITLE
Invalid package reference : kotlinx-coroutines-core-1.3.6

### ISSUE THREAD

**Issue by ahetawal-p** [coil-kt/coil#430]
**Describe the bug**
Upgrading to version 0.11.0 of the library, we are seeing lint errors on our build due to the presence of `kotlinx-coroutines-core/1.3.6`

The lint error we see is this: 
```
/home/jenkins/.gradle/caches/modules-2/files-2.1/org.jetbrains.kotlinx/kotlinx-coroutines-core/1.3.6/401dd0/kotlinx-coroutines-core-1.3.6.jar: 

Error: Invalid package reference in org.jetbrains.kotlinx:kotlinx-coroutines-core; 
not included in Android: java.lang.instrument. 
Referenced from kotlinx.coroutines.debug.AgentPremain. [InvalidPackage]
```

Here is the related bug with 1.3.6 on `kotlinx.coroutines`
https://github.com/Kotlin/kotlinx.coroutines/issues/2004.

Is it possible to upgrade to 1.3.7 to remove this lint error.

**Comment by colinrtwhite** [comment]
If you add `implementation("org.jetbrains.kotlinx:kotlinx-coroutines-android:1.3.7")` to your local build file does it fix the issue? There's some changes in master I need to finish before making a new release.

**Comment by ahetawal-p** [comment]
We already use 1.3.7 in other places, so this dependency is already added.
I have to do this for forcing it:
```
configurations.all {
        resolutionStrategy {
            force "org.jetbrains.kotlinx:kotlinx-coroutines-android:1.3.7"
        }
    }
```

**Comment by colinrtwhite** [comment]
Hmm I can't promise a patch release for this, sorry - but if another issue comes up that requires a patch release I'll update to `1.3.7` as part of it.

**Comment by ahetawal-p** [comment]
No worries, just please include the fix in the upcoming major release. Thank you again.

**Comment by colinrtwhite** [comment]
Not yet - however the next release which updates coroutines to 1.3.8 will be out very soon.

**Comment by colinrtwhite** [comment]
This is now fixed in `0.12.0` by updating Coroutines to `1.3.8`.
